{
    "links": {
        "decline": {
            "href": "https://api.bitbucket.org/2.0/repositories/yt_analysis/yt/pullrequests/460/decline"
        },
        "approve": {
            "href": "https://api.bitbucket.org/2.0/repositories/yt_analysis/yt/pullrequests/460/approve"
        },
        "html": {
            "href": "https://bitbucket.org/yt_analysis/yt/pull-requests/460"
        },
        "activity": {
            "href": "https://api.bitbucket.org/2.0/repositories/yt_analysis/yt/pullrequests/460/activity"
        },
        "self": {
            "href": "https://api.bitbucket.org/2.0/repositories/yt_analysis/yt/pullrequests/460"
        },
        "commits": {
            "href": "https://api.bitbucket.org/2.0/repositories/yt_analysis/yt/pullrequests/460/commits"
        },
        "merge": {
            "href": "https://api.bitbucket.org/2.0/repositories/yt_analysis/yt/pullrequests/460/merge"
        },
        "statuses": {
            "href": "https://api.bitbucket.org/2.0/repositories/yt_analysis/yt/pullrequests/460/statuses"
        },
        "comments": {
            "href": "https://api.bitbucket.org/2.0/repositories/yt_analysis/yt/pullrequests/460/comments"
        },
        "diff": {
            "href": "https://api.bitbucket.org/2.0/repositories/yt_analysis/yt/pullrequests/460/diff"
        }
    },
    "task_count": 0,
    "source": {
        "repository": {
            "full_name": "MatthewTurk/yt",
            "uuid": "{b5dfa70b-707c-4330-92a8-e3ea1bee18b3}",
            "type": "repository",
            "links": {
                "self": {
                    "href": "https://api.bitbucket.org/2.0/repositories/MatthewTurk/yt"
                },
                "html": {
                    "href": "https://bitbucket.org/MatthewTurk/yt"
                },
                "avatar": {
                    "href": "https://bitbucket.org/MatthewTurk/yt/avatar/32/"
                }
            },
            "name": "yt"
        },
        "commit": {
            "hash": "68f53a855cc8",
            "links": {
                "self": {
                    "href": "https://api.bitbucket.org/2.0/repositories/MatthewTurk/yt/commit/68f53a855cc8"
                }
            }
        },
        "branch": {
            "name": "yt"
        }
    },
    "closed_by": {
        "username": "MatthewTurk",
        "type": "user",
        "links": {
            "self": {
                "href": "https://api.bitbucket.org/2.0/users/MatthewTurk"
            },
            "html": {
                "href": "https://bitbucket.org/MatthewTurk/"
            },
            "avatar": {
                "href": "https://bitbucket.org/account/MatthewTurk/avatar/32/"
            }
        },
        "display_name": "Matt Turk",
        "uuid": "{b3f0b280-55fc-4e8a-8bd1-40478d4cbdbf}"
    },
    "title": "First set of Python 3.3 compatibility fixes",
    "id": 460,
    "comment_count": 10,
    "created_on": "2013-03-14T15:06:31.908138+00:00",
    "merge_commit": null,
    "comments": [
        {
            "content": {
                "markup": "markdown",
                "raw": "@xarthisius 's bot has found an issue.  I will update the PR when I fix it.",
                "html": "<p><a href=\"/xarthisius/\" rel=\"nofollow\" title=\"@xarthisius\" class=\"mention\">Kacper Kowalik</a> 's bot has found an issue.  I will update the PR when I fix it.</p>"
            },
            "user": {
                "username": "MatthewTurk",
                "type": "user",
                "links": {
                    "self": {
                        "href": "https://api.bitbucket.org/2.0/users/MatthewTurk"
                    },
                    "html": {
                        "href": "https://bitbucket.org/MatthewTurk/"
                    },
                    "avatar": {
                        "href": "https://bitbucket.org/account/MatthewTurk/avatar/32/"
                    }
                },
                "display_name": "Matt Turk",
                "uuid": "{b3f0b280-55fc-4e8a-8bd1-40478d4cbdbf}"
            },
            "links": {
                "self": {
                    "href": "https://api.bitbucket.org/2.0/repositories/yt_analysis/yt/pullrequests/460/comments/154710"
                },
                "html": {
                    "href": "https://bitbucket.org/yt_analysis/yt/pull-requests/460/_/diff#comment-154710"
                }
            },
            "created_on": "2013-03-14T15:18:20.723360+00:00",
            "updated_on": "2013-03-14T15:18:20.736888+00:00",
            "pullrequest": {
                "type": "pullrequest",
                "links": {
                    "self": {
                        "href": "https://api.bitbucket.org/2.0/repositories/yt_analysis/yt/pullrequests/460"
                    },
                    "html": {
                        "href": "https://bitbucket.org/yt_analysis/yt/pull-requests/460"
                    }
                },
                "title": "First set of Python 3.3 compatibility fixes",
                "id": 460
            },
            "type": "pullrequest_comment",
            "id": 154710
        },
        {
            "content": {
                "markup": "markdown",
                "raw": "tbh I have no idea, it was you who put it [there](https://bitbucket.org/yt_analysis/yt/commits/2cc1aa41184e1447b8398fa1a4252b12a63cb3a7) :)\nHowever, looking at distutils sources it's a **good** idea, since `self.dry_run` is respected there in some places too, whereas some code is run despite it. I think it's safe to assume it's supposed to be run that way.",
                "html": "<p>tbh I have no idea, it was you who put it <a href=\"https://bitbucket.org/yt_analysis/yt/commits/2cc1aa41184e1447b8398fa1a4252b12a63cb3a7\">there</a> :)\nHowever, looking at distutils sources it's a <strong>good</strong> idea, since <code>self.dry_run</code> is respected there in some places too, whereas some code is run despite it. I think it's safe to assume it's supposed to be run that way.</p>"
            },
            "user": {
                "username": "xarthisius",
                "type": "user",
                "links": {
                    "self": {
                        "href": "https://api.bitbucket.org/2.0/users/xarthisius"
                    },
                    "html": {
                        "href": "https://bitbucket.org/xarthisius/"
                    },
                    "avatar": {
                        "href": "https://bitbucket.org/account/xarthisius/avatar/32/"
                    }
                },
                "display_name": "Kacper Kowalik",
                "uuid": "{26af8f1d-35b3-46ce-9939-09915b06e31f}"
            },
            "links": {
                "self": {
                    "href": "https://api.bitbucket.org/2.0/repositories/yt_analysis/yt/pullrequests/460/comments/154747"
                },
                "html": {
                    "href": "https://bitbucket.org/yt_analysis/yt/pull-requests/460/_/diff#comment-154747"
                },
                "code": {
                    "href": "https://api.bitbucket.org/2.0/repositories/yt_analysis/yt/diff/MatthewTurk/yt:53ce98584106..bda5c9c8fe1b?path=setup.py"
                }
            },
            "created_on": "2013-03-14T15:41:46.365651+00:00",
            "updated_on": "2013-03-14T15:41:46.373388+00:00",
            "pullrequest": {
                "type": "pullrequest",
                "links": {
                    "self": {
                        "href": "https://api.bitbucket.org/2.0/repositories/yt_analysis/yt/pullrequests/460"
                    },
                    "html": {
                        "href": "https://bitbucket.org/yt_analysis/yt/pull-requests/460"
                    }
                },
                "title": "First set of Python 3.3 compatibility fixes",
                "id": 460
            },
            "inline": {
                "to": 221,
                "path": "setup.py",
                "from": null
            },
            "type": "pullrequest_comment",
            "id": 154747
        },
        {
            "content": {
                "markup": "markdown",
                "raw": "I think indentation is wrong here, cause now it's outside of the loop",
                "html": "<p>I think indentation is wrong here, cause now it's outside of the loop</p>"
            },
            "user": {
                "username": "xarthisius",
                "type": "user",
                "links": {
                    "self": {
                        "href": "https://api.bitbucket.org/2.0/users/xarthisius"
                    },
                    "html": {
                        "href": "https://bitbucket.org/xarthisius/"
                    },
                    "avatar": {
                        "href": "https://bitbucket.org/account/xarthisius/avatar/32/"
                    }
                },
                "display_name": "Kacper Kowalik",
                "uuid": "{26af8f1d-35b3-46ce-9939-09915b06e31f}"
            },
            "links": {
                "self": {
                    "href": "https://api.bitbucket.org/2.0/repositories/yt_analysis/yt/pullrequests/460/comments/154771"
                },
                "html": {
                    "href": "https://bitbucket.org/yt_analysis/yt/pull-requests/460/_/diff#comment-154771"
                },
                "code": {
                    "href": "https://api.bitbucket.org/2.0/repositories/yt_analysis/yt/diff/MatthewTurk/yt:802fdb182bf2..bda5c9c8fe1b?path=yt%2Fdata_objects%2Fderived_quantities.py"
                }
            },
            "created_on": "2013-03-14T15:53:12.561385+00:00",
            "updated_on": "2013-03-14T15:53:12.569056+00:00",
            "pullrequest": {
                "type": "pullrequest",
                "links": {
                    "self": {
                        "href": "https://api.bitbucket.org/2.0/repositories/yt_analysis/yt/pullrequests/460"
                    },
                    "html": {
                        "href": "https://bitbucket.org/yt_analysis/yt/pull-requests/460"
                    }
                },
                "title": "First set of Python 3.3 compatibility fixes",
                "id": 460
            },
            "inline": {
                "to": 474,
                "path": "yt/data_objects/derived_quantities.py",
                "from": null
            },
            "type": "pullrequest_comment",
            "id": 154771
        },
        {
            "content": {
                "markup": "markdown",
                "raw": "I think at this point it just needs some hand-testing.  I've submitted a pull request to h5py for a fix.",
                "html": "<p>I think at this point it just needs some hand-testing.  I've submitted a pull request to h5py for a fix.</p>"
            },
            "user": {
                "username": "MatthewTurk",
                "type": "user",
                "links": {
                    "self": {
                        "href": "https://api.bitbucket.org/2.0/users/MatthewTurk"
                    },
                    "html": {
                        "href": "https://bitbucket.org/MatthewTurk/"
                    },
                    "avatar": {
                        "href": "https://bitbucket.org/account/MatthewTurk/avatar/32/"
                    }
                },
                "display_name": "Matt Turk",
                "uuid": "{b3f0b280-55fc-4e8a-8bd1-40478d4cbdbf}"
            },
            "links": {
                "self": {
                    "href": "https://api.bitbucket.org/2.0/repositories/yt_analysis/yt/pullrequests/460/comments/155282"
                },
                "html": {
                    "href": "https://bitbucket.org/yt_analysis/yt/pull-requests/460/_/diff#comment-155282"
                }
            },
            "created_on": "2013-03-14T22:15:09.156805+00:00",
            "updated_on": "2013-03-14T22:15:09.164395+00:00",
            "pullrequest": {
                "type": "pullrequest",
                "links": {
                    "self": {
                        "href": "https://api.bitbucket.org/2.0/repositories/yt_analysis/yt/pullrequests/460"
                    },
                    "html": {
                        "href": "https://bitbucket.org/yt_analysis/yt/pull-requests/460"
                    }
                },
                "title": "First set of Python 3.3 compatibility fixes",
                "id": 460
            },
            "type": "pullrequest_comment",
            "id": 155282
        },
        {
            "content": {
                "markup": "markdown",
                "raw": "@MatthewTurk I've made #527 for you. We don't want \"adding python3 support\" to get lost during release announcement creation ;-)",
                "html": "<p><a href=\"/MatthewTurk/\" rel=\"nofollow\" title=\"@MatthewTurk\" class=\"mention\">Matt Turk</a> I've made #527 for you. We don't want \"adding python3 support\" to get lost during release announcement creation ;-)</p>"
            },
            "user": {
                "username": "xarthisius",
                "type": "user",
                "links": {
                    "self": {
                        "href": "https://api.bitbucket.org/2.0/users/xarthisius"
                    },
                    "html": {
                        "href": "https://bitbucket.org/xarthisius/"
                    },
                    "avatar": {
                        "href": "https://bitbucket.org/account/xarthisius/avatar/32/"
                    }
                },
                "display_name": "Kacper Kowalik",
                "uuid": "{26af8f1d-35b3-46ce-9939-09915b06e31f}"
            },
            "links": {
                "self": {
                    "href": "https://api.bitbucket.org/2.0/repositories/yt_analysis/yt/pullrequests/460/comments/155654"
                },
                "html": {
                    "href": "https://bitbucket.org/yt_analysis/yt/pull-requests/460/_/diff#comment-155654"
                }
            },
            "created_on": "2013-03-15T06:12:43.719331+00:00",
            "updated_on": "2013-03-15T06:12:43.732210+00:00",
            "pullrequest": {
                "type": "pullrequest",
                "links": {
                    "self": {
                        "href": "https://api.bitbucket.org/2.0/repositories/yt_analysis/yt/pullrequests/460"
                    },
                    "html": {
                        "href": "https://bitbucket.org/yt_analysis/yt/pull-requests/460"
                    }
                },
                "title": "First set of Python 3.3 compatibility fixes",
                "id": 460
            },
            "type": "pullrequest_comment",
            "id": 155654
        },
        {
            "content": {
                "markup": "markdown",
                "raw": "For what it's worth, this has stalled on figuring out how best to support metaclasses.",
                "html": "<p>For what it's worth, this has stalled on figuring out how best to support metaclasses.</p>"
            },
            "user": {
                "username": "MatthewTurk",
                "type": "user",
                "links": {
                    "self": {
                        "href": "https://api.bitbucket.org/2.0/users/MatthewTurk"
                    },
                    "html": {
                        "href": "https://bitbucket.org/MatthewTurk/"
                    },
                    "avatar": {
                        "href": "https://bitbucket.org/account/MatthewTurk/avatar/32/"
                    }
                },
                "display_name": "Matt Turk",
                "uuid": "{b3f0b280-55fc-4e8a-8bd1-40478d4cbdbf}"
            },
            "links": {
                "self": {
                    "href": "https://api.bitbucket.org/2.0/repositories/yt_analysis/yt/pullrequests/460/comments/197013"
                },
                "html": {
                    "href": "https://bitbucket.org/yt_analysis/yt/pull-requests/460/_/diff#comment-197013"
                }
            },
            "created_on": "2013-04-12T15:41:37.352160+00:00",
            "updated_on": "2013-04-12T15:41:37.358746+00:00",
            "pullrequest": {
                "type": "pullrequest",
                "links": {
                    "self": {
                        "href": "https://api.bitbucket.org/2.0/repositories/yt_analysis/yt/pullrequests/460"
                    },
                    "html": {
                        "href": "https://bitbucket.org/yt_analysis/yt/pull-requests/460"
                    }
                },
                "title": "First set of Python 3.3 compatibility fixes",
                "id": 460
            },
            "type": "pullrequest_comment",
            "id": 197013
        },
        {
            "content": {
                "markup": "markdown",
                "raw": "Just wanted to say that I'm very interested in this pull request since I am now doing production work in Python 3, but I have to switch back to Python 2 for yt. It looks like the latest developer version of h5py now works properly in Python 3.3.",
                "html": "<p>Just wanted to say that I'm very interested in this pull request since I am now doing production work in Python 3, but I have to switch back to Python 2 for yt. It looks like the latest developer version of h5py now works properly in Python 3.3.</p>"
            },
            "user": {
                "username": "astrofrog",
                "type": "user",
                "links": {
                    "self": {
                        "href": "https://api.bitbucket.org/2.0/users/astrofrog"
                    },
                    "html": {
                        "href": "https://bitbucket.org/astrofrog/"
                    },
                    "avatar": {
                        "href": "https://bitbucket.org/account/astrofrog/avatar/32/"
                    }
                },
                "display_name": "astrofrog",
                "uuid": "{bcaaba2f-d294-45d4-850a-00ce6b0d3e54}"
            },
            "links": {
                "self": {
                    "href": "https://api.bitbucket.org/2.0/repositories/yt_analysis/yt/pullrequests/460/comments/370641"
                },
                "html": {
                    "href": "https://bitbucket.org/yt_analysis/yt/pull-requests/460/_/diff#comment-370641"
                }
            },
            "created_on": "2013-07-10T09:12:02.697538+00:00",
            "updated_on": "2013-07-10T09:12:02.841180+00:00",
            "pullrequest": {
                "type": "pullrequest",
                "links": {
                    "self": {
                        "href": "https://api.bitbucket.org/2.0/repositories/yt_analysis/yt/pullrequests/460"
                    },
                    "html": {
                        "href": "https://bitbucket.org/yt_analysis/yt/pull-requests/460"
                    }
                },
                "title": "First set of Python 3.3 compatibility fixes",
                "id": 460
            },
            "type": "pullrequest_comment",
            "id": 370641
        },
        {
            "content": {
                "markup": "markdown",
                "raw": "@astrofrog ah, great news, thanks!  Right now I'm a bit short on time, but I think the main thing left is metaclass conversion.  I'm going to see if I can work on that at some point in the next few weeks, because I'd like to start using Python 3.3 in production as well!  :)",
                "html": "<p><a href=\"/astrofrog/\" rel=\"nofollow\" title=\"@astrofrog\" class=\"mention\">astrofrog</a> ah, great news, thanks!  Right now I'm a bit short on time, but I think the main thing left is metaclass conversion.  I'm going to see if I can work on that at some point in the next few weeks, because I'd like to start using Python 3.3 in production as well!  :)</p>"
            },
            "user": {
                "username": "MatthewTurk",
                "type": "user",
                "links": {
                    "self": {
                        "href": "https://api.bitbucket.org/2.0/users/MatthewTurk"
                    },
                    "html": {
                        "href": "https://bitbucket.org/MatthewTurk/"
                    },
                    "avatar": {
                        "href": "https://bitbucket.org/account/MatthewTurk/avatar/32/"
                    }
                },
                "display_name": "Matt Turk",
                "uuid": "{b3f0b280-55fc-4e8a-8bd1-40478d4cbdbf}"
            },
            "parent": {
                "links": {
                    "self": {
                        "href": "https://api.bitbucket.org/2.0/repositories/yt_analysis/yt/pullrequests/460/comments/370641"
                    },
                    "html": {
                        "href": "https://bitbucket.org/yt_analysis/yt/pull-requests/460/_/diff#comment-370641"
                    }
                },
                "id": 370641
            },
            "created_on": "2013-07-10T10:22:05.147588+00:00",
            "links": {
                "self": {
                    "href": "https://api.bitbucket.org/2.0/repositories/yt_analysis/yt/pullrequests/460/comments/370886"
                },
                "html": {
                    "href": "https://bitbucket.org/yt_analysis/yt/pull-requests/460/_/diff#comment-370886"
                }
            },
            "pullrequest": {
                "type": "pullrequest",
                "links": {
                    "self": {
                        "href": "https://api.bitbucket.org/2.0/repositories/yt_analysis/yt/pullrequests/460"
                    },
                    "html": {
                        "href": "https://bitbucket.org/yt_analysis/yt/pull-requests/460"
                    }
                },
                "title": "First set of Python 3.3 compatibility fixes",
                "id": 460
            },
            "type": "pullrequest_comment",
            "id": 370886,
            "updated_on": "2013-07-10T10:22:05.251772+00:00"
        },
        {
            "content": {
                "markup": "markdown",
                "raw": "With the inclusion of the six module we should be able to make headway on this.  I'll have a shot at it sometime in the next little while, but if it grows too complex we may simply hold off and only make 3.0 applicable to both Py2 and Py3.",
                "html": "<p>With the inclusion of the six module we should be able to make headway on this.  I'll have a shot at it sometime in the next little while, but if it grows too complex we may simply hold off and only make 3.0 applicable to both Py2 and Py3.</p>"
            },
            "user": {
                "username": "MatthewTurk",
                "type": "user",
                "links": {
                    "self": {
                        "href": "https://api.bitbucket.org/2.0/users/MatthewTurk"
                    },
                    "html": {
                        "href": "https://bitbucket.org/MatthewTurk/"
                    },
                    "avatar": {
                        "href": "https://bitbucket.org/account/MatthewTurk/avatar/32/"
                    }
                },
                "display_name": "Matt Turk",
                "uuid": "{b3f0b280-55fc-4e8a-8bd1-40478d4cbdbf}"
            },
            "links": {
                "self": {
                    "href": "https://api.bitbucket.org/2.0/repositories/yt_analysis/yt/pullrequests/460/comments/595823"
                },
                "html": {
                    "href": "https://bitbucket.org/yt_analysis/yt/pull-requests/460/_/diff#comment-595823"
                }
            },
            "created_on": "2013-09-23T16:13:26.500141+00:00",
            "updated_on": "2013-09-23T16:13:26.508456+00:00",
            "pullrequest": {
                "type": "pullrequest",
                "links": {
                    "self": {
                        "href": "https://api.bitbucket.org/2.0/repositories/yt_analysis/yt/pullrequests/460"
                    },
                    "html": {
                        "href": "https://bitbucket.org/yt_analysis/yt/pull-requests/460"
                    }
                },
                "title": "First set of Python 3.3 compatibility fixes",
                "id": 460
            },
            "type": "pullrequest_comment",
            "id": 595823
        },
        {
            "content": {
                "markup": "markdown",
                "raw": "This will go into yt 3.0, and I am declining this pull request.",
                "html": "<p>This will go into yt 3.0, and I am declining this pull request.</p>"
            },
            "user": {
                "username": "MatthewTurk",
                "type": "user",
                "links": {
                    "self": {
                        "href": "https://api.bitbucket.org/2.0/users/MatthewTurk"
                    },
                    "html": {
                        "href": "https://bitbucket.org/MatthewTurk/"
                    },
                    "avatar": {
                        "href": "https://bitbucket.org/account/MatthewTurk/avatar/32/"
                    }
                },
                "display_name": "Matt Turk",
                "uuid": "{b3f0b280-55fc-4e8a-8bd1-40478d4cbdbf}"
            },
            "links": {
                "self": {
                    "href": "https://api.bitbucket.org/2.0/repositories/yt_analysis/yt/pullrequests/460/comments/628323"
                },
                "html": {
                    "href": "https://bitbucket.org/yt_analysis/yt/pull-requests/460/_/diff#comment-628323"
                }
            },
            "created_on": "2013-10-01T19:32:58.276849+00:00",
            "updated_on": "2013-10-01T19:32:58.284650+00:00",
            "pullrequest": {
                "type": "pullrequest",
                "links": {
                    "self": {
                        "href": "https://api.bitbucket.org/2.0/repositories/yt_analysis/yt/pullrequests/460"
                    },
                    "html": {
                        "href": "https://bitbucket.org/yt_analysis/yt/pull-requests/460"
                    }
                },
                "title": "First set of Python 3.3 compatibility fixes",
                "id": 460
            },
            "type": "pullrequest_comment",
            "id": 628323
        }
    ],
    "reason": "Development halted for a while, but not forever!",
    "type": "pullrequest",
    "close_source_branch": false,
    "state": "DECLINED",
    "description": "With these changes, yt will compile and install under Python 3.3.  This is not complete yet, as h5py is now the blocker ( https://groups.google.com/d/msg/h5py/-S1wYiqLQzg/YBbUQFVnezUJ ) but it is enough to import much of yt and all of the C extensions.\r\n\r\nAll of these changes should be compatible with 2.7, but I would like to request testing.  A few notes:\r\n\r\n * I have moved `build_py(self)` outside of the `dry-run` check in `setup.py`.  Is this correct, @xarthisius and @ngoldbaum ?\r\n * I have fixed a number of tab/space issues.  These show up incorrectly in this diff as the tab did not expand to 4 spaces.\r\n * The changes to `png_writer.pyx` may need some examination by someone other than me.\r\n\r\nMostly, moving to Python 3 still mostly has only disadvantages.  However, it will in principle allow using yt from within Blender (blender.org). (ping @jnaiman )\r\n\r\nFor a complete install script to get 3.3:\r\n\r\nhttp://paste.yt-project.org/show/3233/\r\n\r\nThis does not install or build yt.\r\n\r\n*UPDATE 1*: Turning off `use_2to3` which should fix Fido\r\n\r\n*UPDATE 2*: Fixing bug @xarthisius noted.\r\n\r\n*UPDATE 3*: Compiles and imports in Python 3.3.  Before it can be accepted I need to conduct detailed testing on Python 2.7 and Python 3.3.  But I believe this is mostly ready, modulo typos or other minor bugs.",
    "author": {
        "username": "MatthewTurk",
        "type": "user",
        "links": {
            "self": {
                "href": "https://api.bitbucket.org/2.0/users/MatthewTurk"
            },
            "html": {
                "href": "https://bitbucket.org/MatthewTurk/"
            },
            "avatar": {
                "href": "https://bitbucket.org/account/MatthewTurk/avatar/32/"
            }
        },
        "display_name": "Matt Turk",
        "uuid": "{b3f0b280-55fc-4e8a-8bd1-40478d4cbdbf}"
    },
    "updated_on": "2013-10-01T19:33:08.822274+00:00",
    "destination": {
        "repository": {
            "full_name": "yt_analysis/yt",
            "uuid": "{b73cb37c-40f2-493a-aaa9-41bc8d8974cc}",
            "type": "repository",
            "links": {
                "self": {
                    "href": "https://api.bitbucket.org/2.0/repositories/yt_analysis/yt"
                },
                "html": {
                    "href": "https://bitbucket.org/yt_analysis/yt"
                },
                "avatar": {
                    "href": "https://bitbucket.org/yt_analysis/yt/avatar/32/"
                }
            },
            "name": "yt"
        },
        "commit": {
            "hash": "a80979a5173c",
            "links": {
                "self": {
                    "href": "https://api.bitbucket.org/2.0/repositories/yt_analysis/yt/commit/a80979a5173c"
                }
            }
        },
        "branch": {
            "name": "yt"
        }
    }
}